### ISSUE TITLE
Tree component is not accessible with tab key

### ISSUE THREAD

**Issue by ghost** [vaadin/framework#7994]
The tree component is not Section 508 compliant because you cannot tab through the items.. You can add something like this:

`Page.getCurrent().getJavaScript().execute("$(document).ready(function(){$('.v-tree-node-children .v-tree-node, .v-tree').attr('tabindex', '0');});");`

but you still cannot select them with the Enter key.
